src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -102,17 +64,111 @@
  * To configure the check:
  * </p>
  * <pre>
- * &lt;module name=&quot;FallThrough&quot;/&gt;
+ * &lt;module name="FallThrough"/&gt;
  * </pre>
  * <p>
- * or
+ * Example:
+ * </p>
+ * <pre>
+ * public void foo() throws Exception {
+ *   int i = 0;
+ *   while (i &gt;= 0) {
+ *     switch (i) {
+ *       case 1:
+ *         i++;
+ *       case 2: // violation, previous case contains code but lacks
+ *               // break, return, throw or continue statement
+ *         i++;
+ *         break;
+ *       case 3: // OK
+ *         i++;
+ *         return;
+ *       case 4: // OK
+ *         i++;
+ *         throw new Exception();
+ *       case 5: // OK
+ *         i++;
+ *         continue;
+ *       case 6: // OK
+ *       case 7: // OK, previous case does not contain code
+ *         i++;
+ *         // OK, by default the last case is not checked
```

Comment:
`OK, by default the last case might not have 'break'`

Reply:
Besides 'break', it can also have 'return', 'throw', 'continue'. Should I put those in too?
Like this?
`OK, by default the last case might not have 'break', 'return', 'throw' or 'continue'`

Reply:
https://docs.oracle.com/javase/specs/jls/se8/html/jls-14.html#jls-14.15

> A break statement with no label attempts to transfer control to the innermost enclosing switch, while, do, or for statement of the immediately enclosing method or initializer; this statement, which is called the break target, then immediately completes normally.
To be precise, a break statement with no label always completes abruptly, the reason being a break with no label. 

main part is `transfer control to the innermost enclosing statement`

lets do wording like:
`OK, by default the last case might not have statement that transfer control`

Reply:
fixed